docker-compose.yml

```diff
@@ -16,21 +16,15 @@ services:
         hard: 65536 # Hard limit for open files
     image: caddy:latest
     ports:
-      - "80:9944" # Map external 80 to Caddy's 9944 for Node/RPC
-      - "443:9944" # Map external 443 to Caddy's 9945 for HTTPS
-      - "8081:8080" # Map external 8081 to Caddy's 8080 for Hasura
-      - "3011:3001" # Map external 3011 to Caddy's 3001 for Consensus
-      - "3012:3002" # Map external 3012 to Caddy's 3002 for Leaderboard
-      - "3013:3003" # Map external 3013 to Caddy's 3003 for Staking
+      - "8000:9944" # Map external 8000
     volumes:
       - ./indexers/Caddyfile:/etc/caddy/Caddyfile # Mount the Caddyfile
       - caddy_data:/data # Volume for Let's Encrypt certificates
       - caddy_config:/config
-
-    environment:
-      - DOMAIN=subql.${NETWORK_ID}.subspace.network
     depends_on:
       - node
+      - hasura
```

Comment:
If I read this correctly, the only thing Caddy manage now is the node (looking the the exposed port, so why does it depend on hasura and postgres

Reply:
You are right, corrected that.